transcom/mymove
# B-18851 FRA Update screen to reflect value in cents, not dollars

## [Agility Ticket](https://www13.v1host.com/USTRANSCOM38/assetdetail.v1?number=B-18851)

_See Fig2 below for the format you should see the FRA in_
(For instance, if the FRA is 135.4 cents, it should read 135.4 in stead of $1.35)

### Test steps I took to verify success
1. Use the "HHGMoveInSITEndsToday" option from the Test harness list.
2. Copy the move locator from the JSON object displayed (see Fig1 below)
3. Navigate back to the sign in page for milmove office users
4. Sign in as a Multi-role user under the KKFA GBLOC.
5. Start by clicking the Sign in button for a Prime user.
6. Search for the move by the move locator you copied earlier. Then click on that move in the list.
7. Click the "Create payment request" button.
8. From the list of service items you are able to select from you should see an item called "Domestic origin SIT fuel surcharge". Put a check in the checkbox for that item ("Add to payment request").
9. Click the "Submit payment request" button at the bottom of the form.
10. Click the "Change user role" link at the top of the page.
11. Make the appropriate selection to log in as a TOO user.
12. Search for the same move locator and click on the associated move.
13. Click the "Payment requests" tab.
Note: there is a known issue when certain conditions are met that the TOO must review weights associated. Steps 14-15 are to workaround this known issue.
14. Click the "Review Weights" button.
15. Click the "Payment requests" tab again.
16. Click the "Review Service Items" button.
17. Observe that the added service item for fuel surcharge has a "Show calculations" link. Click that link to show the calculation details.
18. The FRA value should be displayed in Cents to the tenths place decimal.

## Fig1
![Screenshot 2024-03-25 at 2 13 29 PM](https://github.com/transcom/mymove/assets/147739091/e18e2206-9755-4d71-9a4a-8e99127cd97b)

## Fig2
![Screenshot 2024-03-27 at 2 00 18 PM](https://github.com/transcom/mymove/assets/147739091/5ba241be-a347-49a4-ab9a-762750ce98f0)
